components/x-follow-button/src/FollowButton.jsx

```diff
@@ -0,0 +1,74 @@
+import { h } from '@financial-times/x-engine';
+import { withActions } from '@financial-times/x-interaction';
+
+import Button from './Button';
+
+import styles from './styles/main.scss';
+
+const followButtonActions = withActions((props) => ({
+	onSubmitAction(event) {
+		event.preventDefault();
+		console.log('the follow button was pressed');
+		console.log('the default event was prevented');
+		console.log(`the action suppose to trigger the submission of the form with "${event.target.method}" method and "${event.target.action}" action`);
+	},
+	onClickAction(event) {
+		return ({isSelected}) => ({
+			isSelected: !isSelected
+		});
+	}
+}));
+
+const getFormAction = (conceptId, followPlusDigestEmail, isSelected) => {
+	if (followPlusDigestEmail) {
+		return `/__myft/api/core/follow-plus-digest-email/${conceptId}?method=put`
+	} else if (isSelected) {
+		return `/__myft/api/core/followed/concept/${conceptId}?method=delete`
+	} else {
+		return `/__myft/api/core/followed/concept/${conceptId}?method=put`
+	}
+};
+
+const BaseFollowButton = ({
+	conceptId,
+	csrfToken,
+	followPlusDigestEmail,
+	isSelected,
+	extraButtonClasses,
+	variant,
+	alternateText,
+	buttonText,
+	actions,
+	name
+}) => (
+	<form
+		className="{{extraClasses}}"
```

Comment:
this `"{{extraClasses}}"` looks left over from the handlebars version, should it be `{extraClasses}` instead (and included in the props)?

Reply:
yes, that is correct! Will fix it

Reply:
(sorry 😬 )